webapp/components/survey/CategoryDetails/CategoryDetails.js

```diff
@@ -15,34 +15,30 @@ import { useAuthCanEditSurvey } from '@webapp/store/user'
 import { FormItem, Input } from '@webapp/components/form/input'
 import UploadButton from '@webapp/components/form/uploadButton'
 
-import CategoryImportSummary from './components/categoryImportSummary'
-import LevelEdit from './components/levelEdit'
+import ImportSummary from './ImportSummary'
+import LevelEdit from './LevelEdit'
 
-import * as CategoryState from './categoryState'
-import * as Actions from './actions'
+import * as Actions from '../../../loggedin/surveyViews/category/actions'
```

Comment:
I assume you won't need this anymore.. you can remove it in the next pull request

Reply:
sure